### PR TITLE
Changed header menu, logo added, fixed main layout behavior

### PR DESCRIPTION
**Pull request by AlexanderBazukevich**

### CODE REVIEW

**Review by Zairbekov — APPROVED** [review]

File: src/app/shared/custom-components/logo/logo.component.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<div class="logo"></div>
+<!-- <mat-icon svgIcon="logo"></mat-icon> -->
```
Comment (abarmina) [inline src/app/shared/custom-components/logo/logo.component.html]: we need the logo to be a link. maybe try something like this? and without background and @media

```
<a href="">
    <picture>
        <source srcset="img/icon-mobile.svg" media="(max-width: 768px)" >
        <img src="img/icon-desktop.svg" alt="">
    </picture>
</a>
```

File: src/app/shared/custom-components/logo/logo.component.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.logo {
+  cursor: pointer;
+  height: 35px;
+  width: 200px;
+  background-image: url(../../../../assets/images/logo.svg);
```
Comment (abarmina) [inline src/app/shared/custom-components/logo/logo.component.scss:5]: don't we need an absolute path?

File: src/app/pages/home-page/home-page.component.scss [hunk 3]
```diff
@@ -8,34 +8,43 @@
...
-    padding: 10px;
-    transition: margin 0.3s linear;
-    &_hidden {
-      margin-left: -100%;
-      transition: margin 0.3s linear;
-    }
-  }
-
-  @media screen and (max-width: 768px) {
+  @media screen and (max-width: 1023px) {
```
Comment (abarmina) [inline src/app/pages/home-page/home-page.component.scss:25]: I suggest creating variables for media query values

File: src/app/navigation/header/header.component.html [hunk 4]
```diff
@@ -1,8 +1,33 @@
 <div class="header">
-  <button mat-button routerLink="home">Home</button>
-  <button mat-button routerLink="admin">Admin</button>
-  <button mat-button routerLink="login">Sing-in</button>
-  <button mat-icon-button (click)="openSidebar()">
-    <mat-icon>person</mat-icon>
-  </button>
+  <mat-toolbar>
+    <app-logo routerLink="home" (click)="adminPageVisible = false"></app-logo>
+    <span class="header__spacer"></span>
```
Comment (abarmina) [inline src/app/navigation/header/header.component.html]: can we use ```justify-content: space-between```?

File: src/app/pages/home-page/home-page.component.ts [hunk 5]
```diff
@@ -6,5 +6,9 @@ import { Component } from '@angular/core';
   styleUrls: ['./home-page.component.scss']
 })
 export class HomePageComponent {
-  isMapVisible = false;
+  isListVisible = true;
+
+  toggleListView(): void {
+    this.isListVisible = this.isListVisible ? false : true;
```
Comment (abarmina) [inline src/app/pages/home-page/home-page.component.ts]: ```this.isListVisible = !this.isListVisible```

File: src/app/navigation/header/header.component.html [hunk 6]
```diff
@@ -1,8 +1,33 @@
...
+    <nav class="header__nav">
+      <button
+        class="button button_l"
+        mat-button
+        *ngIf="!adminPageVisible; else homePageButton"
+        routerLink="admin"
+        (click)="toggleAdminPage()"
+      >
+        Manage
+      </button>
```
Comment (abarmina) [inline src/app/navigation/header/header.component.html]: it's a new page, it should be a link ```<a>```

File: src/app/navigation/header/header.component.html (hunk 4 above)
Comment (AlexanderBazukevich) [reply src/app/navigation/header/header.component.html]: it's material toolbar implementation

File: src/app/pages/home-page/home-page.component.ts (hunk 5 above)
Comment (AlexanderBazukevich) [reply src/app/pages/home-page/home-page.component.ts]: Sure)

File: src/app/shared/custom-components/logo/logo.component.html (hunk 1 above)
Comment (AlexanderBazukevich) [reply src/app/shared/custom-components/logo/logo.component.html]: Removed logo component. Made link. But I left image loading as background.

File: src/app/pages/home-page/home-page.component.scss (hunk 3 above)
Comment (AlexanderBazukevich) [reply src/app/pages/home-page/home-page.component.scss:25]: I agree, but suggest make separate commit for this

File: src/assets/images/logo.svg [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+</svg>
```
Comment (tlazarchuk) [inline src/assets/images/logo.svg:9]: Why you have a lot of paths?

File: src/app/styles/components/buttons.scss [hunk 8]
```diff
@@ -0,0 +1,19 @@
+.button.button {
+  &_icon {
+    &:focus {
+      outline: none;
+    }
+    &::ng-deep .mat-button-focus-overlay {
+      opacity: 0;
+    }
+  }
+}
```
Comment (tlazarchuk) [inline src/app/styles/components/buttons.scss:11]: If you use some difficult construction, you should leave a comment

**Review by tlazarchuk — APPROVED** [review]

File: src/app/navigation/header/header.component.html (hunk 4 above)
Comment (abarmina) [reply src/app/navigation/header/header.component.html]: ok)

File: src/assets/images/logo.svg (hunk 7 above)
Comment (AlexanderBazukevich) [reply src/assets/images/logo.svg:9]: It`s default svg logo from official exadel web-site.

File: src/app/styles/components/buttons.scss (hunk 8 above)
Comment (AlexanderBazukevich) [reply src/app/styles/components/buttons.scss:11]: Ок) Will add.

File: src/app/navigation/header/header.component.html (hunk 6 above)
Comment (AlexanderBazukevich) [reply src/app/navigation/header/header.component.html]: I`m not sure of this. Here I use material-buttons, they are already well styled.

Comment (abarmina) [reply src/app/navigation/header/header.component.html]: ```<a mat-raised-button href="#">Link</a>```

https://material.angular.io/components/button/overview

**Pull request merged**